doc/library/config.txt

```diff
@@ -716,6 +716,23 @@ import theano and print the config variable, as in:
 
     Location of the magma library.
 
+.. attribute:: config.ctc.enabled
+
+    Default: ``'False'``
+
+    If ``'True'``, use `warp-ctc <https://github.com/baidu-research/warp-ctc>`_ for CTC
+    loss computations.
+
+    If ``'False'``, disable warp-ctc.
+
+.. attribute:: config.ctc.root
+
+    Default: ``''``
+
+    Location of the warp-ctc folder. The folder should contain a build
+    subfolder with the shared library (libwarpctc.so), and another
```

Comment:
It can also support having lib and lib64 as include directories.

The Implementation was done this way because the build directory is the one created with the default instructions from the [warp-ctc](https://github.com/baidu-research/warp-ctc/blob/master/README.md#compilation) repository.